support-frontend/assets/pages/digital-subscription-landing/components/productBlock.jsx

```diff
@@ -115,15 +110,22 @@ const appImage = (
 
 type ProductCardPropTypes = {
   title: string,
-  subtitle: Node,
+  subtitle: string,
+  shortSubTitle: string | null,
```

Comment:
You could make the `ProductCard` prop `shortSubTitle` an optional `string` rather than `| null`. This would save having to pass `null` as a prop when there's no sub title.

Reply:
Sure. Do you mean `shortSubTitle?: string` or `shortSubTitle: Option<string>`?

Reply:
`shortSubTitle?: string` looks good

Reply:
👍